lib/automatiek/gem.rb

```diff
@@ -43,11 +43,15 @@ def files
       Dir.glob("#{vendor_lib}/**/*.rb")
     end
 
+    def require_entrypoint
```

Comment:
maybe make this an overridable attribute, with this as the default?

Reply:
Pushed 2c614e0fa3fb4962b57b7c2b05a0713e4e799f78 to try address this. Is that what you meant?